planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/structs/data.hpp

```diff
@@ -128,6 +128,22 @@ struct Info
     terminal_lane_changing_velocity = _lc_metrics.velocity;
     shift_line = _shift_line;
   }
+
+  void set_prepare(const PhaseMetrics & _prep_metrics)
+  {
+    longitudinal_acceleration.prepare = _prep_metrics.actual_lon_accel;
+    velocity.prepare = _prep_metrics.velocity;
+    duration.prepare = _prep_metrics.duration;
+    length.prepare = _prep_metrics.length;
+  }
+
+  void set_lane_changing(const PhaseMetrics & _lc_metrics)
+  {
+    longitudinal_acceleration.prepare = _lc_metrics.actual_lon_accel;
+    velocity.prepare = _lc_metrics.velocity;
+    duration.prepare = _lc_metrics.duration;
+    length.prepare = _lc_metrics.length;
+  }
```

Comment:
`_lc_metrics` is set to prepare. Is this intended?
Also what is the purpose of the front underscore again?

Reply:
good catch

Reply:
fixed in [1a4fd5b](https://github.com/autowarefoundation/autoware.universe/pull/9592/commits/1a4fd5b54ddc1fb7e148d4bb662ad82681f23709)